lib/widgets/atoms/reminder_banner.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:flutter/material.dart';
+import 'package:mm_flutter_app/constants/app_constants.dart';
+
+class ReminderBanner extends StatelessWidget {
+  final String titleText;
+  final String subtitleText;
+  final String ctaText;
+
+  const ReminderBanner({
+    Key? key,
+    required this.titleText,
+    required this.subtitleText,
+    required this.ctaText
+  }) : super(key: key);
+
+  Widget _buildTextColumn(context){
+    return Expanded(
+      flex: 1,
+      child: Column(
+          crossAxisAlignment:  CrossAxisAlignment.start,
+          mainAxisSize: MainAxisSize.min,
+          children: <Widget> [
+              Text(
+                titleText,
+                style: TextStyles.cardTitle(context),
+                textAlign: TextAlign.start,
+                softWrap: true
+              ),
+              const SizedBox(height: Insets.widgetSmallestInset),
+              Text(
+                subtitleText,
+                style: TextStyles.cardSubtitle(context),
+                textAlign: TextAlign.start,
+                softWrap: true
+              ),
+          ],
+        )
+      );
+  }
+
+  Widget _buildImageAndButtonColumn(context){
+    return Expanded(
+      flex: 1,
+      child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: <Widget> [
+            const SizedBox(
+                child : Image(
+                image: AssetImage("assets/images/StockImage1.png"),
+                height: Dimensions.reminderBannerImageHeight,
+            )),
+            InkWell(
+            onTap:  () => {
+              debugPrint("hi")
+            },
+            child: Row(children: [
```

Comment:
The text and icon in this Row are aligned to the start of the row, whereas the AssetImage is always centered. This means that as the screen size increases the image and the text become misaligned. I think the easiest fix is to add MainAxisAlignment.center to the Row, but you can experiment with other approaches

Reply:
Good catch, fixed